### DISCUSSION TITLE
git-blame.nvim Doesn't Show Up

### DISCUSSION THREAD

**Discussion by poita66** [LunarVim/LunarVim#2595] [Q&A] [answered]
I installed f-person/git-blame.nvim as per the instructions on the LunarVim docs: https://www.lunarvim.org/plugins/02-extra-plugins.html#git-blame

And I can't figure out why it's not showing up. I've tried setting `lvim.lsp.diagnostics.virtual_text = false` in-case that was interfering.
I have no idea how to diagnose this either, so any help is greatly appreciated

**Comment by f-person** [comment, marked as answer]
Hello, @poita66. The docs, for some reason, set `vim.g.gitblame_enabled` to 0, which disables the plugin (functions like `:GitBlameOpenCommitURL` and `GitBlameCopySHA` will still work though). Try setting `vim.g.gitblame_enabled` to 1 and see if that helps :)

```lua
{
  "f-person/git-blame.nvim",
  event = "BufRead",
  config = function()
    vim.cmd "highlight default link gitblame SpecialComment"
    vim.g.gitblame_enabled = 1 -- Change the value to 1 here
  end,
},
```
